### PR TITLE
feature/combined -> master

### PR DESCRIPTION
**Pull request by mzroth**
This change is [Reviewable](https://reviewable.io/reviews/byu-oit/terraform-aws-postman-test-lambda/23)

### CODE REVIEW

**Review by yoshutch — COMMENTED** [review]
I'm not exactly sure what this is trying to do, but it looks good.

You'll need to update the README and examples to have a new semantic version

**Comment by mzroth** [comment]
I updated the versions in the examples. I will cut a new release once we merge to master. I went with a minor version update because there shouldn't be any breaking changes, but since it was a shift in way the module could function it seemed like more than a patch.

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ You can provide a postman collection and environment to be tested in one of two
 1. Provided in your github repo
     ```hcl
     module "postman_test_lambda" {
-      source = "github.com/byu-oit/terraform-aws-postman-test-lambda?ref=v2.3.1"
+      source = "github.com/byu-oit/terraform-aws-postman-test-lambda?ref=v2.4.1"
```
Comment (yoshutch) [inline README.md]: why not just `2.4.0`?

**Review by yoshutch — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (mzroth) [reply README.md]: Because sometimes I don't use my brain. 😂 Should be fixed now

**Pull request merged**